dev/com.ibm.ws.security.token.ltpa_fat/fat/src/com/ibm/ws/security/token/ltpa/fat/LTPAInitializationVectorTests.java

```diff
@@ -0,0 +1,1476 @@
+     * 2. Replace LTPA keys with validation keys from respective servers to ensure no validation keys exist in either server.
+     * 3. Access a simple servlet with form login using valid credentials on Server #1
+     * 4. authentication should be successful and retrieve the SSO cookie
+     * 5. Attempt to access the simple servlet with form login on Server #2 using the SSO cookie from Server #1
+     * 6. Authentication fails because the SSO token cannot be decrypted by the primary key of server 2
+     * (passwords might be the same but since the primary key values are different and there are no validation keys containing ltpa1, it fails). IV is set to
+     * different valid ltpa keys
+     */
+
+    @Mode(TestMode.LITE)
+    @Test
+    @AllowedFFDC({ "javax.crypto.BadPaddingException", "java.lang.IllegalArgumentException" })
+    public void testLTPAValidationKeyUsage_no_validation_keys() throws Exception {
+        // Copy valid ltpa keys to each server, the ltpa keys are configured using different keysPassword
+        copyFileToServerResourcesSecurityDir(ALT_VALIDATION_KEY1_PATH, server1);
+        copyFileToServerResourcesSecurityDir(ALT_VALIDATION_KEY2_PATH, server2);
+
+        // Configure both servers, and replace the generated LTPA keys with the known valid ltpa key in server 1
+        configureServer("true", "10", true, server1);
+        renameFileIfExists(VALIDATION_KEY1_PATH, DEFAULT_KEY_PATH, true, server1);
+        // Wait for the LTPA configuration to be ready after the change
+        assertNotNull("Expected LTPA configuration ready message not found in the log.",
+                      server1.waitForStringInLog("CWWKS4107A", 5000));
+
+        configureServer("true", "10", true, server2);
+        // Configure both servers, and replace the generated LTPA keys with the known valid ltpa key in server 2
+        renameFileIfExists(VALIDATION_KEY2_PATH, DEFAULT_KEY_PATH, true, server2);
+        // Wait for the LTPA configuration to be ready after the change
+        assertNotNull("Expected LTPA configuration ready message not found in the log.",
+                      server2.waitForStringInLog("CWWKS4105I", 5000));
+
+        // Change the default keysPassword to that of the added ltpa keys file (Liberty)
+        ServerConfiguration serverConfig = server1.getServerConfiguration();
+        LTPA ltpa = serverConfig.getLTPA();
+        setLTPAKeyPasswordElement(ltpa, "{xor}Lz4sLCgwLTs=");
+        // Wait for the LTPA configuration to be ready after the change
+        assertNotNull("Expected LTPA configuration ready message not found in the log.",
+                      server1.waitForStringInLog("CWWKS4107A", 5000));
+        updateConfigDynamically(server1, serverConfig);
+
+        // Initial login to simple servlet for form login1
+        String response1 = server1FlClient1.accessProtectedServletWithAuthorizedCredentials(FormLoginClient.PROTECTED_SIMPLE, validUser, validPassword);
```

Comment:
I"m not convinced we need to do any of this. The passwords in server.xml are already the same so couldn't we just use the keys which are created by server start and skip copying keys into the server, renaming them etc.?